File: LinearSearch.py
#LinearSearch
def LinearSearch(x,l):
    #Go through array and looking for "x"
    for a in range(len(l)):
        if l[a]==x:
            return a
    return False

#Create ourList and Check LinearSearch
ourList=[3,64,7,23,62,12,34]
print(LinearSearch(4,ourList))#False
print(LinearSearch(7,ourList))#2

File: LinearSearch.js
//LinearSearch
let LinearSearch=(x,l)=>{
    //Go through array and looking for our "x"
    for(let i=0;i<l.length;i++){
        if(x==l[i]){
            return i
        }
    }
    return false    
}

//Create ourList and Check LinearSearch
ourList=[3,54,12,523,64,57]
console.log(LinearSearch(56,ourList))//False
console.log(LinearSearch(12,ourList))//2